### PR TITLE
Extending DEX-AMM to add account/position data

### PR DESCRIPTION
**Pull request by this-username-is-taken**

### CODE REVIEW

File: schema-dex-amm-extended.graphql [hunk 1]
```diff
@@ -611,62 +650,182 @@ type LiquidityPoolHourlySnapshot @entity {
...
+  outputTokenBalance: BigInt
+
+  " Total reward token accumulated under this position, in native amounts "
+  cumulativeRewardTokenAmounts: [BigInt!]
+
+  " Number of deposits related to this position "
+  depositCount: Int!
+
+  " All deposit events of this position "
+  deposits: [Deposit!]! @derivedFrom(field: "position")
```
Comment (nemani) [inline schema-dex-amm-extended.graphql:739]: There is no field `position` in `Withdraw` and `Deposit`. Needs to be updated.

File: schema-dex-amm-extended.graphql [hunk 2]
```diff
@@ -167,9 +167,21 @@ interface Protocol {
   " Number of cumulative unique users "
   cumulativeUniqueUsers: Int!
 
+  " Number of cumulative liquidity providers "
```
Comment (nemani) [inline schema-dex-amm-extended.graphql:170]: These 4 fields should also be in `DexAmmProtocol` which implements interface `Protocol`

File: schema-dex-amm-extended.graphql (hunk 1 above)
Comment (this-username-is-taken) [reply schema-dex-amm-extended.graphql:739]: updated

File: schema-dex-amm-extended.graphql (hunk 2 above)
Comment (this-username-is-taken) [reply schema-dex-amm-extended.graphql]: updated

**Pull request merged**